src/firebase/config.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  apiKey: process.env.VUE_APP_FIREBASE_API_KEY,
+  //authDomain: "ss-amble.firebaseapp.com",
+  databaseURL: process.env.VUE_APP_FIREBASE_DATABASE_URL,
+  projectId: process.env.VUE_APP_FIREBASE_PROJECT_ID
+  //storageBucket: "ss-amble.appspot.com"
+  //messagingSenderId: "750012918172",
+  //appId: "1:750012918172:web:b33c9a0198c2ae9cf0c059"
+};
```

Comment:
Firebase config in a tidy bundle

The whole object gets imported. To use it elsewhere, we have to import it first as a const, then call it within that component